From a02347043677bb5218bdd4483fefae891da352d8 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Fri, 24 Jan 2025 22:15:29 +0100
Subject: [PATCH] =?UTF-8?q?user-record:=20rename=20USER=5FDISPOSITION=5FMA?=
 =?UTF-8?q?SK=5FMAX=20=E2=86=92=20USER=5FDISPOSITION=5FMASK=5FALL?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

On request by yuwata.

(cherry picked from commit 27cce1f1efb564f30563eb0e3d37264bed3b093c)

Related: RHEL-143036
---
 src/shared/user-record.h | 2 +-
 src/userdb/userdbctl.c   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/shared/user-record.h b/src/shared/user-record.h
index a188dc0d0b..5ecc443d22 100644
--- a/src/shared/user-record.h
+++ b/src/shared/user-record.h
@@ -486,7 +486,7 @@ typedef struct UserDBMatch {
         };
 } UserDBMatch;
 
-#define USER_DISPOSITION_MASK_MAX ((UINT64_C(1) << _USER_DISPOSITION_MAX) - UINT64_C(1))
+#define USER_DISPOSITION_MASK_ALL ((UINT64_C(1) << _USER_DISPOSITION_MAX) - UINT64_C(1))
 
 bool user_name_fuzzy_match(const char *names[], size_t n_names, char **matches);
 int user_record_match(UserRecord *u, const UserDBMatch *match);
diff --git a/src/userdb/userdbctl.c b/src/userdb/userdbctl.c
index 4525de3a46..2c3b274828 100644
--- a/src/userdb/userdbctl.c
+++ b/src/userdb/userdbctl.c
@@ -1510,7 +1510,7 @@ static int parse_argv(int argc, char *argv[]) {
 
         /* If not mask was specified, use the all bits on mask */
         if (arg_disposition_mask == UINT64_MAX)
-                arg_disposition_mask = USER_DISPOSITION_MASK_MAX;
+                arg_disposition_mask = USER_DISPOSITION_MASK_ALL;
 
         if (arg_from_file)
                 arg_boundaries = false;
